incubator/binding-mqtt-kafka/src/main/java/io/aklivity/zilla/runtime/binding/mqtt/kafka/internal/stream/MqttKafkaSessionFactory.java

```diff
@@ -3451,6 +3682,7 @@ private MessageConsumer newGroupStream(
         final KafkaBeginExFW kafkaBeginEx =
             kafkaBeginExRW.wrap(writeBuffer, BeginFW.FIELD_OFFSET_EXTENSION, writeBuffer.capacity())
                 .typeId(kafkaTypeId)
+                // TODO: this will be rejected if if sessionExpiryMs > max group timeout in Kafka
```

Comment:
As mentioned above, this will not be rejected, reply `KafkaGroupBeginEx` will provide a value in the supported range.